packages/database/src/migrations/20210113032005-SamoaHeatmapHomeVillageOfConfirmedCases-modifies-data.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function deleteItemByOneCondition(db, table, condition) {
+  const [key, value] = Object.entries(condition)[0];
+  return db.runSql(`
+    DELETE FROM
+      "${table}"
+    WHERE
+      "${key}" = '${value}';
+  `);
+}
+const indicator = {
+  id: generateId(),
+  code: 'COVID_SAMOA_HEATMAP_CONFIRMED_CASES',
+  builder: 'arithmetic',
+  config: {
+    formula:
+      '(WS_COVID19_Clinsurv_75 == 0 | WS_COVID19_Clinsurv_137 == 1 | WS_COVID19_Clinsurv_79 == 0 | (WS_COVID19_Clinsurv_83 != 3 & WS_COVID19_Clinsurv_83 != 4) | WS_COVID19_Clinsurv_87 == 0) ? 1 : 0',
```

Comment:
Are you sure doing an `|` is the right kind of `or` here? According to the mathjs docs this is a bitwise or, which I believe resolves to a number rather than a `true/false` value. I suspect it will work fine with this conditional, but perhaps it may be safer and read better using a logic or, eg:
```
(WS_COVID19_Clinsurv_75 == 0 OR WS_COVID19_Clinsurv_137 == 1 OR WS_COVID19_Clinsurv_79 == 0 OR (WS_COVID19_Clinsurv_83 != 3 AND WS_COVID19_Clinsurv_83 != 4) OR WS_COVID19_Clinsurv_87 == 0) ? 1 : 0
```

I'm okay with the bitwise or if it works though, especially if the performance is better, I just think it might be risky if someone copied this without realising

Reply:
Thanks Rohan, for performance purpose I agree to switch to 'OR'.